docs: add 5.23.4 changelog (#52646)
ant-design/ant-design: CHANGELOG.en-US.md

---

### 5.23.4

> `2025-02-03`

Reviewer: ```suggestion
`2025-02-04`
```

> `2025-02-04`

Reviewer: ```suggestion
`2025-02-05`
```

First release in the Year of the Snake, wishing you a prosperous start! 🐍

> - ⌨️ Fixed `Pagination` accessibility issue by supplementing missing ARIA attributes support. [#52616](https://github.com/ant-design/ant-design/pull/52616) [@aojunhao123](https://github.com/aojunhao123)  

Reviewer 2: 组件名不要加 `

> - 🐞 Added `TextArea` component support in `Space.Compact`.[#52639](https://github.com/ant-design/ant-design/pull/52639) [@Can-Chen](https://github.com/Can-Chen)  

Reviewer 2: ```suggestion
- 🐞 Added `TextArea` component support in `Space.Compact`. [#52639](https://github.com/ant-design/ant-design/pull/52639) [@Can-Chen](https://github.com/Can-Chen)  
```

> - 🐞 Fixed identical text/background color issue in horizontal `Menu` with `theme="dark"`.[#52617](https://github.com/ant-design/ant-design/pull/52617) [@afc163](https://github.com/afc163)  

Reviewer 2: ```suggestion
- 🐞 Fixed Menu with `theme="dark"` and `mode="horizontal"` identical text/background color issue.[#52617](https://github.com/ant-design/ant-design/pull/52617) [@afc163](https://github.com/afc163)  
```

> - 🐞 Clarified misleading description of `Input.Search`'s `enterButton` property regarding its behavior and interactions.[#52614](https://github.com/ant-design/ant-design/pull/52614) [@Muhammad-Taif-Khan](https://github.com/Muhammad-Taif-Khan)  

Reviewer 2: ```suggestion
- 🐞 Clarified misleading description of Input.Search's `enterButton` property regarding its behavior and interactions.[#52614](https://github.com/ant-design/ant-design/pull/52614) [@Muhammad-Taif-Khan](https://github.com/Muhammad-Taif-Khan)  
```
